### DISCUSSION TITLE
Group by ?

### DISCUSSION THREAD

**Discussion by davidlygagnon** [nick-keller/react-datasheet-grid#322] [General]
Hi, 

I was looking at the library and wondering if there's a way to elegantly do group by that would not involve creating multiple individual data sheet grid components? Ideally, you could insert a new row header that specify the group and that row could have a different height than the actual data rows.

Thanks,
David

**Comment by nick-keller** [comment]
Here is an example: https://react-datasheet-grid.netlify.app/docs/examples/collapsible-rows
